src/envo/envo-edit.owl

```diff
@@ -37862,6 +38067,1591 @@ AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.obolibra
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_01002132> (zooplankton abundance)
+
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_01002132> "Ecological abundance which inheres in a collection of zooplankters.")
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/ENVO_01002132> <https://orcid.org/0000-0002-4366-3088>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.obolibrary.org/obo/ENVO_01002132> "2024-12-16T17:26:18Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_01002132> "zooplankton abundance"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_01002132> <http://purl.obolibrary.org/obo/ENVO_01002119>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_01002133> (variable about dioxygen concentration in ocean water)
+
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_01002133> "A variable about the concentration of dioxygen in ocean water.")
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/ENVO_01002133> <https://orcid.org/0000-0002-4366-3088>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.obolibrary.org/obo/ENVO_01002133> "2024-12-17T20:03:02Z"^^xsd:dateTime)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <http://purl.obolibrary.org/obo/ENVO_01002133> <https://goosocean.org/document/17473>)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasRelatedSynonym> <http://purl.obolibrary.org/obo/ENVO_01002133> "variable about dioxygen dissolved in marine water")
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasRelatedSynonym> <http://purl.obolibrary.org/obo/ENVO_01002133> "variable about dioxygen dissolved in ocean water")
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#inSubset> <http://purl.obolibrary.org/obo/ENVO_01002133> <http://purl.obolibrary.org/obo/envo#envoEOVs>)
+AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/ENVO_01002133> "This class is intended for use in labelling data aligned to the IOC UNESCO Global Ocean Observing System's Essential Ocean Variable framework. For more, see https://book.odis.org/thematics/variables/")
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_01002133> "variable about dioxygen concentration in ocean water"@en)
+EquivalentClasses(<http://purl.obolibrary.org/obo/ENVO_01002133> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/ENVO_01001996> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/ENVO_01001998> <http://purl.obolibrary.org/obo/ENVO_3100060>)))
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_01002133> <http://purl.obolibrary.org/obo/ENVO_01001996>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_01002134> (macronutrient)
```

Comment:
is this intentional (macronutrient is-a variable)? Note also CHEBI has claimed the label "macronutrient" for a role (and yes this is problematic on multiple levels but label clashes cause multiple downstream problems)